### PR TITLE
[keyboard]Move kb83 keyboard.

### PR DESCRIPTION
**Pull request by jonylee1986**
## Description
move hfdkb/keyboard_sw/k83 --> inland/kb83

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: data/mappings/keyboard_aliases.hjson [hunk 1]
```diff
@@ -868,6 +868,9 @@
     "infinity60": {
         "target": "input_club/infinity60"
     },
+    "hfdkb/keyboard_sw/k83":{
+        "target":"inland/kb83"
```
Comment (fauxpark) [inline data/mappings/keyboard_aliases.hjson:872]:
```suggestion
        "target": "inland/kb83"
```

File: data/mappings/keyboard_aliases.hjson [hunk 2]
```diff
@@ -868,6 +868,9 @@
     "infinity60": {
         "target": "input_club/infinity60"
     },
+    "hfdkb/keyboard_sw/k83":{
```
Comment (fauxpark) [inline data/mappings/keyboard_aliases.hjson:871]: Should be moved below `hecomi/alpha`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Would like for this to stay unmerged until I can get a fix in for the Configurator layout data (which hopefully shouldn't take long):

image

**Comment by noroadsleft** [comment]
> a fix in for the Configurator layout data

#21297

**Comment by jonylee1986** [comment]
Sorry, I misunderstood your meaning. I have also submitted changes here, and I am not sure if this will lead to any conflicts. If there are any, please follow your modifications. Your modifications are very suitable for this keyboard.@noroadsleft

**Comment by noroadsleft** [comment]
@jonylee1986,

Sorry for the lack of clarity. I requested a hold on this PR because QMK Configurator is using this data already, but if this PR gets merged, the Configurator implementation won't get fixed until the next `develop`->`master` merge at the end of August.

Once my PR is merged, ideally I will be able to integrate its changes into this PR from my end and you won't need to do anything.

**Comment by noroadsleft** [comment]
@jonylee1986,

My Configurator layout fix has been merged to `qmk:master`, and the implementation now appears as:


Unfortunately, it appears either my attempt or yours to fix the layout data have created a merge conflict.

Removing the last six commits from your GitHub fork's `move_kb83` branch by running:

```
git push --force-with-lease git@github.com:jonylee1986/qmk_firmware_master.git c085c54765:move_kb83
```

in your command line interface should fix the conflict.

**Review by noroadsleft — APPROVED** [review]
... or you could merge the branches together - which strangely didn't merge cleanly when I tried it locally. :confused: 

Looks good, thanks!

**Pull request merged**